### PR TITLE
Update docs for mongodb package-names

### PR DESCRIPTION
**Pull request by radovanradic**
Issue related to this has been reported several times, one of them [this one](https://github.com/micronaut-projects/micronaut-data/issues/2623), so just updating docs with this information which will hopefully reduce number of issues.

### CODE REVIEW

**Comment by radovanradic** [comment]
Reopening to trigger license/cla check that was stuck

File: src/main/docs/guide/mongo/mongoQuickStart.adoc [hunk 1]
```diff
@@ -113,6 +113,15 @@ You can now perform CRUD (Create, Read, Update, Delete) operations on the entity
 
 snippet::example.BookRepositorySpec[project-base="doc-examples/mongo-example", tags="inject", indent="0"]
```
Comment (radovanradic) [inline src/main/docs/guide/mongo/mongoQuickStart.adoc:115]: @dstepanov Does this look ok?

Comment (dstepanov) [reply src/main/docs/guide/mongo/mongoQuickStart.adoc:115]: Can you please move it above after the first mention of the configuration?

**Review by dstepanov — CHANGES_REQUESTED** [review]

File: src/main/docs/guide/mongo/mongoQuickStart.adoc (hunk 1 above)
Comment (dstepanov) [reply src/main/docs/guide/mongo/mongoQuickStart.adoc:115]: Did you push?

Comment (radovanradic) [reply src/main/docs/guide/mongo/mongoQuickStart.adoc]: I did now, sorry. I was on vpn and maybe it didn't get published.

**Review by dstepanov — APPROVED** [review]

**Pull request merged**